common/utils.js

```diff
@@ -454,7 +454,9 @@ const getItsEdgeContract = (chainConfig) => {
 };
 
 const getItsEdgeChains = (config, excludeChainName) => {
-    return Object.keys(config.chains).filter((chain) => getItsEdgeContract(config.chains[chain]) && chain !== excludeChainName);
+    return Object.keys(config.chains)
+        .filter((chain) => config.chains[chain].contracts?.InterchainTokenService?.address)
+        .filter((chain) => getItsEdgeContract(config.chains[chain]) && chain !== excludeChainName);
```

Comment:
```suggestion
        .filter((chain) => getItsEdgeContract(config.chains[chain]) && chain !== excludeChainName);
```
this is incorrect. `getItsEdgeContract` should be used instead, since it handles Stellar specific contract naming as well, that the new filter doesn't consider. Modify the helper to not throw an error, and let the caller do it? Or create a non-exported helper that doesn't throw an error and gets used by these two functions?